README.md

```diff
@@ -525,24 +602,45 @@ def bar(param: <param type>) -> <return type>:
 ```
 
 If the code is type-annotated most IDEs can statically validate it for you
-although you might need to install appropriate extensions. You can also validate
-types in CLI. For example given (see [`./types.py`](./types.py)):
-
-```python
-def foo(a: str) -> bytes:
-    return True
-
-
-foo(1)
-```
-
-`mypy` will produce:
+although you might need to install appropriate extensions. You can also
+validate types in CLI. For example, given the included
+[`sample_types.py`](./sample_types.py), `mypy` will produce:
```

Comment:
renamed `types.py` to `sample_types.py` because my python interpreter didn't like that `types.py` shadows the existing built-in `types` module.

removed inlining of the file content here to de-duplicate code.